controllers/productController.js

```diff
@@ -0,0 +1,30 @@
+const productService = require("../services/productService");
+
+const getproducts = async (req, res) => {
+  try {
+    const { categoryId, sort } = req.query;
+    const lists = await productService.getproducts(categoryId, sort);
+    res.status(200).json({ data: lists });
+    return lists;
```

Comment:
해당 함수에서 response를 보내고 난 후에 `lists`를 return하는 이유가 있나요? 
```suggestion
    return res.status(200).json({ data: lists });
```